images-list

```diff
@@ -34,3 +34,8 @@ us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-extern
 openpolicyagent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.11
 k8s.gcr.io/k8s-dns-node-cache rancher/k8s-dns-node-cache 1.15.7
 sonobuoy/sonobuoy rancher/sonobuoy-sonobuoy v0.16.3
+banzaicloud/logging-operator rancher/logging-operator 3.4.0
```

Comment:
We like to prepend the original org to the image name -> `rancher/banzaicloud-logging-operator`
Also, any image that doesn't currently exist in our dockerhub will need to get a repo set up for it, I can give you details. Ideally that would get checked on PR CI checks, made issue for that.